### PR TITLE
SCP-4090 add topic, cluster and loglevel for custom code logging output

### PR DESCRIPTION
**Pull request by byang-confluent**
Release Notes
-------------

Breaking Changes
- PLACEHOLDER

New Features
- PLACEHOLDER

Bug Fixes
- PLACEHOLDER

Checklist
---------
- [ ] Leave this box unchecked if features are not yet available in production

What
----

add topic, cluster and loglevel for custom code logging output

References
----------

Test & Review
-------------

### CODE REVIEW

**Comment by brianstrauch** [comment]
> If this PR introduces any user-facing changes, please document them below. Please delete any unused section titles and placeholders.
Please match the style of previous release notes: https://docs.confluent.io/confluent-cli/current/release-notes.html

File: test/fixtures/output/custom-code-logging/create-no-env.golden [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-| Region      | us-west-2  |
-| Environment | env-000000 |
-+-------------+------------+
++-------------+-------------+
+| Id          | ccl-123456  |
+| Cloud       | AWS         |
+| Region      | us-west-2   |
+| Environment | env-000000  |
+| Topic       | topic-123   |
+| Cluster     | cluster-123 |
```
Comment (brianstrauch) [inline test/fixtures/output/custom-code-logging/create-no-env.golden:7]: Does cluster refer to a Kafka cluster? Normally that would start with "lkc-".

Comment (byang-confluent) [reply test/fixtures/output/custom-code-logging/create-no-env.golden:7]: yes, it's kafka cluster. do you suggest to change this name?

**Comment by byang-confluent** [comment]
> If this PR introduces any user-facing changes, please document them below. Please delete any unused section titles and placeholders.

@brianstrauch , this cli is released in main yesterday. and we don't have any customer yet.

**Comment by brianstrauch** [comment]
I see you added your PR description to the "Release Notes" section. Maybe you meant to add it to the "What" section?

File: test/fixtures/output/custom-code-logging/create-no-env.golden [hunk 2]
```diff
@@ -1,6 +1,9 @@
-+-------------+------------+
-| Id          | ccl-123456 |
-| Cloud       | AWS        |
-| Region      | us-west-2  |
-| Environment | env-000000 |
-+-------------+------------+
++-------------+-------------+
+| Id          | ccl-123456  |
```
Comment (brianstrauch) [inline test/fixtures/output/custom-code-logging/create-no-env.golden:2]:
```suggestion
| ID          | ccl-123456  |
```

**Review by brianstrauch — CHANGES_REQUESTED** [review]

File: internal/custom-code-logging/command_custom_code_logging_create.go [hunk 3]
```diff
@@ -107,6 +107,6 @@ func (c *customCodeLoggingCommand) createCustomCodeLogging(cmd *cobra.Command, a
 	}
 	table := output.NewTable(cmd)
 	table.Add(getCustomCodeLogging(resp))
-	table.Filter([]string{"Id", "Cloud", "Region", "Environment"})
+	table.Filter([]string{"Id", "Cloud", "Region", "Environment", "Topic", "Cluster", "LogLevel"})
```
Comment (brianstrauch) [inline internal/custom-code-logging/command_custom_code_logging_create.go:110]:
```suggestion
	table.Filter([]string{"ID", "Cloud", "Region", "Environment", "Topic", "Cluster", "LogLevel"})
```

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**